frontend/packages/core/src/card.tsx

```diff
@@ -51,23 +49,63 @@ const StyledCardHeaderContainer = styled.div({
   background: "#EBEDFB",
 });
 
-interface CardHeaderProps extends Pick<MuiCardHeaderProps, "avatar" | "title"> {
+const StyledCardHeader = styled(Grid)({
+  padding: "16px",
+  minHeight: "72px",
+});
+
+const StyledCardHeaderAvater = styled.div({
+  marginRight: "16px",
+  fontSize: "24px",
+});
+
+const StyledDivider = styled(Divider)({
+  color: "rgba(13, 16, 48, 0.38)",
+  height: "36px",
+  alignSelf: "center",
+});
+
+const StyledGridItem = styled(Grid)({
+  textAlign: "center",
+});
+
+export interface CardHeaderSections {
+  title: React.ReactNode;
+  subheader?: React.ReactNode;
+}
+
+interface CardHeaderProps {
+  avatar: React.ReactNode;
   children?: React.ReactNode;
+  sections?: CardHeaderSections[];
+  title: React.ReactNode;
 }
 
-const CardHeader = ({ avatar, children, title }: CardHeaderProps) => (
-  <StyledCardHeaderContainer>
-    <MuiCardHeader
-      style={{
-        padding: "16px",
-      }}
-      disableTypography
-      avatar={avatar}
-      title={<StyledTypography variant="h3">{title}</StyledTypography>}
-    />
-    {children}
-  </StyledCardHeaderContainer>
-);
+// TODO: do we want to move the sections into the Dash card and keep this core card with just avatar and title
```

Comment:
I think it makes sense to keep this here. It's similar to having actions off the header. If anything I think it might be good to restructure the files here and separate out the card parts into various files but that's a larger discussion :)

Reply:
yeah good idea. added a TODO